### DISCUSSION TITLE
When does the class name from style function returns undefined?

### DISCUSSION THREAD

**Discussion by tawhidurrahman** [vanilla-extract-css/vanilla-extract#387] [Q&A]
I'm getting class name of a style from another file undefined when referenced in a selector.
Can't share any reproducible link or actual the repo.
![Capture](https://user-images.githubusercontent.com/24841912/134960281-88b7b113-124e-4c83-b2bf-c19d5aeb91d6.PNG)

**Comment by tmm** [comment]
Can you post your code for nav? It's possible you are using a selector/syntax that isn't defined/supported.
